packages/nimbus/src/components/search-input/search-input.recipe.tsx

```diff
@@ -0,0 +1,160 @@
+import { defineSlotRecipe } from "@chakra-ui/react/styled-system";
+
+/**
+ * Recipe configuration for the SearchInput component.
+ * Defines the styling variants and base styles using Chakra UI's slot recipe system.
+ */
+export const searchInputSlotRecipe = defineSlotRecipe({
+  slots: ["root", "leadingElement", "input", "clearButton"],
+  // Unique class name prefix for the component
+  className: "nimbus-search-input",
+
+  // Base styles applied to all instances of the component
+  base: {
+    root: {
+      display: "inline-flex",
+      position: "relative",
+      cursor: "text",
+      borderRadius: "200",
+      boxShadow: "inset 0 0 0 var(--border-width) var(--border-color)",
+      alignItems: "center",
+      _focusWithin: {
+        layerStyle: "focusRing",
+      },
+
+      _hover: {
+        backgroundColor: "primary.2",
+      },
+
+      "&[data-invalid='true']": {
+        "--border-color": "colors.critical.7",
+        "--border-width": "sizes.50",
+        color: "critical.11",
+      },
+
+      _disabled: {
+        bg: "neutral.3",
+        layerStyle: "disabled",
+      },
+    },
+
+    leadingElement: {
+      display: "flex",
+      alignItems: "center",
+      color: "neutral.11",
+      pointerEvents: "none",
+    },
+
+    input: {
+      cursor: "inherit",
+      display: "block",
+      flexGrow: 1,
+      bg: "transparent",
+      outline: "none",
+      fontSize: "inherit",
+
+      _placeholder: {
+        opacity: 0.5,
+        color: "currentColor",
+      },
+
+      "&::-webkit-search-cancel-button": {
+        display: "none",
+      },
+
+      "&::-webkit-search-decoration": {
+        display: "none",
+      },
+    },
+
+    clearButton: {
```

Comment:
@jaikamat the `Select` and `Combobox` components both use an `IconButton` (e.g. [here](https://github.com/commercetools/nimbus/blob/2cd30955d9a23d40887fd568dbde36afdeb31125/packages/nimbus/src/components/combobox/components/combobox.single-select-button-group.tsx#L21)) to render the clear button.

Perhaps it would make more sense to do so here as well, both for consistency and for the sake of not duplicating styles?

Reply:
Thanks @ByronDWall, see 5ff2ceb0c1c6064e1e551b7b6f1f6eecd2c23ef0, should fix it